lib/cmd.js

```diff
@@ -105,6 +105,10 @@ class Cmd {
         _options.logLevel = _options.loglevel; // fix casing
         _options.onlyCompile = _options.contracts;
         _options.client = _options.client || 'geth';
+        embark.initConfig(env || 'development', {
```

Comment:
`initConfig` has been resurrected from earlier hasn't it? My issue with this is that `initConfig` initialises `Events`, `Logger`, and `Config` objects along with the config routine that loads the config files, but those same objects/routines are also initialised and run in the constructor of `Engine`. So we have two different sources of code runnings that do similar things. What does `initConfig` do that the `Engine` constructor does differently? Maybe it would be best to pass an option to the `Engine` constructor instead to avoid running both of these.